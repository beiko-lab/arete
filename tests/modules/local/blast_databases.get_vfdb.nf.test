nextflow_process {

    name "Test Process GET_VFDB"
    script "modules/local/blast_databases.nf"
    process "GET_VFDB"

    test("Can download VFDB") {

        then {
            assert process.success
            assert process.trace.tasks().size() == 1
            assert snapshot(process.out).match()
            with(process.out) {
              // check if emitted output has been created
              assert vfdb.size() == 1
            }
        }

    }

}
