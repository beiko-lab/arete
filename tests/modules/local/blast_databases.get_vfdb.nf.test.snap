{
    "Should run without failures": {
        "content": [
            {
                "vfdb": [
                    "VFDB_setA_pro.fas.gz:md5,4ba35bcde2914956aa9d1deb9d732bde"
                ]
            }
        ],
        "timestamp": "2023-02-03T14:29:23+0000"
    },
    "Can download VFDB": {
        "content": [
            {
                "vfdb": [
                    "VFDB_setA_pro.fas.gz:md5,4ba35bcde2914956aa9d1deb9d732bde"
                ]
            }
        ],
        "timestamp": "2023-02-03T14:29:23+0000"
    }
}